src/terrain/terrain.cpp

```diff
@@ -612,16 +612,25 @@ Terrain::get_path_type(int xs, int ys, int zs, int xf, int yf, int zf) const {
     uint8_t y_diff = abs(ys - yf);
     uint8_t z_diff = abs(zs - zf);
 
+    if (x_diff > 1 || y_diff > 1 || z_diff > 1){
+        LOG_ERROR(logging::terrain_logger, "diff is greater than one!!");
```

Comment:
What diff? Give better error messages than this. And include some objects

Reply:
diff is difference, the error message will be removed.